src/New.js

```diff
@@ -41,7 +41,7 @@ class New extends Component {
 }
 
 New.propTypes = {
-  /** Defines the deep properties used in the component. Ex: foo.bar */
+  /** It must be equal to the reducer name. Defines the deep properties used in the component. Ex: foo.bar */
```

Comment:
```suggestion
  /** It must be equal to the reducer name defined in your `rootReducer.js`. It can use dot notation for deep properties used in the component. Ex: foo.bar */
```